src/random_test.cpp

```diff
@@ -455,6 +457,21 @@ std::string Column::rand_value() {
   return "";
 }
 
+/* prepare single quoted string for LIKE clause */
+std::string Table::prepare_string(int where, std::string str) {
```

Comment:
would be easy to understand prepare_like_string

Reply:
Done